--- kio-5.4.0/KF5KIOConfig.cmake.in~	2014-11-02 23:58:22.000000000 +0100
+++ kio-5.4.0/KF5KIOConfig.cmake.in	2014-12-09 23:11:42.847217061 +0100
@@ -2,8 +2,8 @@
 
 # Any changes in this ".cmake" file will be overwritten by CMake, the source is the ".cmake.in" file.
 
-set(KCookieServer_DBUS_INTERFACE "${PACKAGE_PREFIX_DIR}/@DBUS_INTERFACES_INSTALL_DIR@/kf5_org.kde.KCookieServer.xml")
-set(KSlaveLauncher_DBUS_INTERFACE "${PACKAGE_PREFIX_DIR}/@DBUS_INTERFACES_INSTALL_DIR@/kf5_org.kde.KSlaveLauncher.xml")
+set(KCookieServer_DBUS_INTERFACE "@DBUS_INTERFACES_INSTALL_DIR@/kf5_org.kde.KCookieServer.xml")
+set(KSlaveLauncher_DBUS_INTERFACE "@DBUS_INTERFACES_INSTALL_DIR@/kf5_org.kde.KSlaveLauncher.xml")
 
 find_dependency(KF5Bookmarks "@KF5_DEP_VERSION@")
 find_dependency(KF5Completion "@KF5_DEP_VERSION@")
